app/verify-auth/route.js

```diff
@@ -87,11 +99,13 @@ export default Route.extend(VerifyAuth, {
       }
     }
 
-    if ( code && get(params, 'login') ) {
-      if ( github.stateMatches(get(params, 'state')) ) {
-        let ghProvider = get(this, 'access.providers').findBy('id', 'github');
+    if ( code && get(params, 'login') || get(params, 'state').includes('login') ) {
```

Comment:
As part of the service consolidation we talked about in review, lets see if we can get the Github state/login param to work the same way as google.

Reply:
Absolutely. I will consolidate both github and google services.